### PR TITLE
Simplify Chinese Translation of Scala Tour: Package Objects

### PR DESCRIPTION
**Pull request by realwunan**
Simplify Chinese Translation of Scala Tour: Package Objects

### CODE REVIEW

File: _zh-cn/tour/package-objects.md [hunk 1]
```diff
@@ -11,7 +11,63 @@ num: 36
...
+Scala 提供包对象作为在整个包中方便的共享使用的容器。
+
+包对象中可以定义任何内容，而不仅仅是变量和方法。 例如，包对象经常用于保存包级作用域的类型别名和隐式转换。 包对象甚至可以继承 Scala 的类和特质。
+
+按照惯例，包对象的代码通常放在名为 `package.scala` 的源文件中。
+
+每个包都允许有一个包对象。 在包对象中的任何定义都被认为是包自身的成员。
+
+看下例。 首先假设包中有一个类 `Fruit` 和三个 `Fruit` 对象
+`gardening.fruits`:
```
Comment (liufengyun) [inline _zh-cn/tour/package-objects.md]: 假设~包中~有一个..对象 __在包__ `gardening.fruits`中？

File: _zh-cn/tour/package-objects.md [hunk 2]
```diff
@@ -11,7 +11,63 @@ num: 36
...
+package gardening
+package object fruits {
+  val planted = List(Apple, Plum, Banana)
+  def showFruit(fruit: Fruit): Unit = {
+    println(s"${fruit.name}s are ${fruit.color}")
+  }
+}
+```
+
+下例演示了如何使用，下例中的对象 `PrintPlanted` 用导入类 `Fruit` 相同的方式来导入 `planted` 和 `showFruit`，在导入包 `gardening.fruits` 时使用通配符：
```
Comment (liufengyun) [inline _zh-cn/tour/package-objects.md]: `下例演示了如何使用` -> `作为一个使用范例`?

File: _zh-cn/tour/package-objects.md (hunk 1 above)
Comment (realwunan) [reply _zh-cn/tour/package-objects.md]: 比原文好一些，thanks a lot.

File: _zh-cn/tour/package-objects.md (hunk 2 above)
Comment (realwunan) [reply _zh-cn/tour/package-objects.md]: that's much better, thanks a lot.

**Comment by realwunan** [comment]
Thanks, @liufengyun , just rebase to remove useless commits

**Comment by liufengyun** [comment]
Thanks a lot @realwunan 👍

**Pull request merged**